### ISSUE TITLE
Invalid argument supplied for foreach()

### ISSUE THREAD

**Issue by JLNNN** [svewap/realurl_404_multilingual#3]
I get this error in
/var/www/htdocs/xxx/typo3conf/ext/realurl_404_multilingual/Classes/Hooks/FrontendHook.php line 112.

Fixed it as following:
Wrap line 112 - 122 with this code:
```php
if (isset($typo3_conf_var_realurl['preVars']) && !empty($typo3_conf_var_realurl['preVars'])) {
// line 112 - 122
}
```

**Comment by JLNNN** [comment]
This problem is now (v1.0.4) occurring in /realurl_404_multilingual/Classes/Hooks/FrontendHook.php line 185. Please fix this 👍

**Comment by svewap** [comment]
Fixed in version 1.0.6
